README.md

```diff
@@ -6,18 +6,38 @@
 </div>
 
 Documentation:
-[![version 1][docs-img]](https://docs.julialang.org)
+[![Documentation (version 1)][docs-img]][docs-url]
 
-Code coverage:
-[![coveralls][coveralls-img]](https://coveralls.io/r/JuliaLang/julia?branch=master)
-[![codecov][codecov-img]](https://codecov.io/github/JuliaLang/julia?branch=master)
+[docs-img]: https://img.shields.io/badge/docs-v1-blue.svg "Documentation (version 1)"
+[docs-url]: https://docs.julialang.org
+
+Continuous integration:
+[![Build status (LTS)][buildkite-lts-img]][buildkite-lts-url]
+[![Build status (stable)][buildkite-stable-img]][buildkite-stable-url]
+[![Build status (master)][buildkite-master-img]][buildkite-master-url]
+[![Build status (scheduled)][buildkite-scheduled-master-img]][buildkite-scheduled-master-url]
```

Comment:
I wonder if, instead, we should just have a single build status, but on `master` branch it points to `buildkite-master-url`, and on 1.6, it points to its own.

Reply:
I was thinking it's nice to be able to go to the README and see all of the statuses quickly, instead of needing to go to each branch individually.

Reply:
I don't think one branch should be advertising the build status of other branches; it makes it look like `master` is broken, when really some other branch is the one that is broken.

Reply:
Alright, I've made PRs so that each branch only advertises its own build status.
- `master`: this PR (#41596)
- `release-1.6`: #41608 
- `release-1.7`: #41609